news/4341.internal

```diff
@@ -0,0 +1 @@
+Fixed the issue #4341 @ajmalwali7
```

Comment:
A news item should include what you changed, as well as referring to the issue number. This helps people who read the change/history log. See examples: https://6.docs.plone.org/volto/release-notes/index.html

Reply:
Done sir, You can check it out.